e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -270,6 +270,174 @@ describe('Desktop notifications', () => {
             });
         });
     });
+
+    it('MM-T491 - Channel notifications: No desktop notification when in focus', () => {
+        cy.apiCreateUser().then(({user}) => {
+            cy.apiAddUserToTeam(testTeam.id, user.id);
+            cy.apiLogin(user);
+
+            cy.apiGetChannelByName(testTeam.name, 'Off-Topic').then(({channel}) => {
+                const message = '/echo test 3';
+
+                // # Go to Off topic
+                cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+                spyNotificationAs('withNotification', 'granted');
+
+                // Have another user send a post with delay
+                cy.postMessageAs({sender: testUser, message, channelId: channel.id});
+
+                // * Desktop notification is not received
+                cy.get('@withNotification').should('not.have.been.called');
+            });
+        });
+    });
+
+    it('MM-T491 - Channel notifications: No desktop notification when in focus', () => {
+        cy.apiCreateUser().then(({user}) => {
+            cy.apiAddUserToTeam(testTeam.id, user.id);
+            cy.apiLogin(user);
+
+            cy.apiGetChannelByName(testTeam.name, 'Off-Topic').then(({channel}) => {
+                const message = '/echo test 3';
+
+                // # Go to Off topic
+                cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+                spyNotificationAs('withNotification', 'granted');
+
+                // Have another user send a post with delay
+                cy.postMessageAs({sender: testUser, message, channelId: channel.id});
+
+                // * Desktop notification is not received
+                cy.get('@withNotification').should('not.have.been.called');
+            });
+        });
+    });
```

Comment:
is this duplicated? or did I miss the difference?

Also, I'm missing MM-T493

Reply:
haha yes there is a dup but for MM-T493 I left a comment in the ticket: 


'Test case MM-T493 cannot be tested currently because we cannot post a message from another user in another team at the moment.'

Reply:
Yeah, I think the limitation for MM-T493 would be the inability to click the desktop notification.